crates/re_viewport/src/viewport.rs

```diff
@@ -350,18 +350,13 @@ impl<'a, 'b> Viewport<'a, 'b> {
             .values()
             .chain(already_added.iter())
         {
-            if existing_view.space_origin == space_view_candidate.space_origin {
+            if existing_view.class_identifier() == space_view_candidate.class_identifier() {
```

Comment:
that would mean that the moment all space views of a given type are edited, we add new ones again? seems very problematic
testing this a bit

Reply:
ah. I misread that. This says if the query of _any_ space view of this type was edited we'll no longer automatically add space views.
A bit aggressive, but I think that's ok